app/index.html

```diff
@@ -371,6 +371,25 @@
                 </div>
             </div>
         </div>
+
+        <div id="send-to-remote-confirmation-modal" class="modal fade" tabindex="-1" role="dialog" aria-hidden="true">
+            <div class="modal-dialog">
+                <div class="modal-content">
+                    <div class="modal-header">
+                        <button type="button" class="close" data-dismiss="modal" aria-hidden="true">×</button>
+                        <h3>Send selected Workflows to a remote scheduler</h3>
+                    </div>
+                    <div class="modal-body" style="height:100%">
+                        <table style="width:100%"><tr><td>Please enter the remote Scheduler<br>URL (http://xxx.xxx.xxx.xxx:port)</td><td><input type="text" id="remoteCatalogURL"></td></tr>
+                        <tr><td>Please enter the id of the bucket<br>of the remote scheduler</td><td><input type="text" id="remoteCatalogBucketId"></td></tr></table>
```

Comment:
This sentence is a little bit strange to me, if I know nothing about workflow-catalog.
Is it better to change the second "of" to "on"?